Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -0,0 +1,269 @@
+
+    /**
+     * Controls whether control should use system default provided focus rects
+     *
+     * @platform windows
+     */
+    disableSystemFocusVisuals: PropTypes.bool,
+
+    /**
+     * Called when the view receives focus.
+     *
+     * @platform windows
+     */
+    onFocus: PropTypes.func,
+
+    /**
+     * Called when the view focus is lost.
+     *
+     * @platform windows
+     */
+    onBlur: PropTypes.func,
+
+    /**
+     * Set of keys that should be handled on key down by this component.
+     *
+     * @platform windows
+     */
+    handledKeyDownKeys: PropTypes.arrayOf(PropTypes.number),
+
+    /**
+     * Set of keys that should be handled on key up by this component.
+     *
+     * @platform windows
+     */
+    handledKeyUpKeys: PropTypes.arrayOf(PropTypes.number),
+
+    /**
+     * Called when key down while component has focus.
+     *
+     * @platform windows
+     */
+    onKeyDown: PropTypes.func,
+
+    /**
+     * Called when key up while component has focus.
+     *
+     * @platform windows
+     */
+    onKeyUp: PropTypes.func,
+  };
+
+  static keys = UIManager.WindowsControl.Constants.Keys;
+
+  /**
+   * Splits styles into two sets: one applying to focusable view, and the other one to the embedded child
+   * The only style we care about is "transform"
+   */
+  static splitStyle(styles) {
+      var sets = {};
+
+      sets.focusableStyle = {};
+      sets.childStyle = {};
+
+      if (styles) {
+        var flattenedStyles = flattenStyle(styles);
+
+        for (var styleName in flattenedStyles) {
+          if (styleName == 'transform') {
+            sets.focusableStyle[styleName] = flattenedStyles[styleName];
+          } else {
+            sets.childStyle[styleName] = flattenedStyles[styleName];
+          }
+        }
+      }
+
+      return sets;
+  }
+
+  /**
+   * Splits native properties used in setNativeProps into two sets: one applying to focusable view, and the other one to
+   * the embedded child
+   * The only style we care about is "transform"
+   */
+  static splitNativeProps(nativeProps) {
+    var sets = {};
+
+    sets.focusableProps = undefined;
+    sets.childProps = nativeProps;
+
+    if (nativeProps && nativeProps.style && 'transform' in nativeProps.style) {
+
+      sets.focusableProps = {};
+      sets.focusableProps.style = {};
+      var childStyle = {};
+
+      var atLeastOneChildProp = false;
+      // There is no need to flatten the styles
+      for (var styleName in nativeProps.style) {
+        if (styleName == 'transform') {
+          sets.focusableProps.style[styleName] = nativeProps.style[styleName];
+        } else {
+          childStyle[styleName] = nativeProps.style[styleName];
+          atLeastOneChildProp = true;
+        }
+      }
+
+      sets.childProps.style = childStyle;
+
+      if (!atLeastOneChildProp) {
+        // Check presence of any other prop
+        for (var propName in sets.childProps) {
+          if (propName != 'style') {
```

Comment:
>!= [](start = 23, length = 2)

`!==` is preferred over `!=` due because `!=` has complicated coercing rules.

Reply:
fixed

---
In reply to: [156484649](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156484649) [](ancestors = 156484649)